src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -112,6 +133,10 @@ $card-left-offset: 40px;
         margin-top: 5px;
     }
 
+    .bcs-comment-container {
+        margin-bottom: 8px;
```

Comment:
If we stick to margin-top for spacing, it makes it easier to compose more elements.